### PR TITLE
:cat: Replace npm with yarn

### LINKED ISSUE #283: Replace npm with yarn

### PR DESCRIPTION
**Pull request by st3v3nhunt**
Fixes #283

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployment of connecting-to-services-pr-284 in nhsuk-dev succeeded (http://connecting-to-services-pr-284.dev.beta.nhschoices.net)

File: package.json [hunk 1]
```diff
@@ -12,23 +12,23 @@
     "lint": "eslint --ext .js,.json .",
     "test": "NODE_ENV=test mocha --recursive test",
     "build-css": "./css-build",
-    "watch-dev": "nodemon app.js | ./node_modules/.bin/bunyan & nodemon -e scss -x npm run build-css -- compact",
-    "watch-dev-debug": "NODE_ENV=development && npm run watch-dev",
+    "watch-dev": "nodemon app.js | ./node_modules/.bin/bunyan & nodemon -e scss -x yarn run build-css -- compact",
```
Comment (markysoft) [inline package.json]: I'm getting the error:
./node_modules/.bin/bunyan: not found

**Comment by c2s-dev** [comment]
:rocket: deployment of connecting-to-services-pr-284 in nhsuk-dev succeeded (http://connecting-to-services-pr-284.dev.beta.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of connecting-to-services-pr-284 in nhsuk-dev succeeded (http://connecting-to-services-pr-284.dev.beta.nhschoices.net)

**Pull request merged**